test/sample-test.ts

```diff
@@ -258,4 +258,14 @@ describe("Starknet", function () {
     expect(latestBlock.block_number).to.be.greaterThan(blockByHash.block_number);
   });
 
+  it("Should update time for following block", async function () {
```

Comment:
Since sample-test.ts is executed on plugin's alpha-goerli tests, this test case should be in another file, not here.